### PR TITLE
ci(backend): Add Supabase

### LINKED ISSUE #8122: Set up Supabase in CI
We need Supabase available in CI to test the now Supabase-dependent backend.

### PR DESCRIPTION
**Pull request by Pwuts**
- Resolves #8122

### Changes 🏗️

- Set up Supabase in the Backend CI
  - Remove separate Postgres DB setup
- Remove unused MinIO service
- Remove multi-platform stuff

### CODE REVIEW

File: .github/workflows/platform-backend-ci.yml [hunk 1]
```diff
@@ -102,25 +68,26 @@ jobs:
...
 
       - name: Generate Prisma Client
         run: poetry run prisma generate
 
+      - id: supabase
+        name: Start Supabase
+        working-directory: .
+        run: |
+          supabase init
+          supabase start --exclude postgres-meta,realtime,storage-api,imgproxy,inbucket,studio,edge-runtime,logflare,vector,supavisor
```
Comment (Pwuts) [inline .github/workflows/platform-backend-ci.yml:82]: @ntindle are you saying we do need `vector`?

**Comment by aarushik93** [comment]
@Pwuts have you ran this already to test?

**Comment by Pwuts** [comment]
image

**Comment by aarushik93** [comment]
Ah cool, my bad i missed that it was already in the actions!

**Review by aarushik93 — APPROVED** [review]

File: .github/workflows/platform-backend-ci.yml (hunk 1 above)
Comment (Pwuts) [reply .github/workflows/platform-backend-ci.yml:82]: We don't need it to pass current CI, so I'm going to merge.

**Pull request merged**